pkg/gardenlet/controller/shoot/seed_registration_control.go

```diff
@@ -676,6 +676,7 @@ func deployGardenlet(ctx context.Context, gardenClient, seedClient, shootedSeedC
 					"seedClientConnection":  externalConfig.SeedClientConnection.ClientConnectionConfiguration,
 					"shootClientConnection": externalConfig.ShootClientConnection,
 					"controllers":           externalConfig.Controllers,
+					"resources":             externalConfig.Resources,
```

Comment:
Can you make this overwriteable from the shooted seed configuration annotation?

Reply:
Done, including tests